cmd/duffle/main.go

```diff
@@ -111,7 +112,46 @@ func prepareDriver(driverName string) (driver.Driver, error) {
 		configurable.SetConfig(driverCfg)
 	}
 
-	return driverImpl, err
+	rm, err := loadRelMapping(relMap)
+	if err != nil {
+		return nil, err
+	}
+
+	// wrap the driver so any relocation mapping is mounted
+	return &driverWithRelocationMapping{
+		driver:     driverImpl,
+		relMapping: rm,
+	}, nil
+}
+
+type driverWithRelocationMapping struct {
+	driver     driver.Driver
+	relMapping string
+}
+
+func (d *driverWithRelocationMapping) Run(op *driver.Operation) error {
+	// if there is a relocation mapping, ensure it is mounted
+	if d.relMapping != "" {
+		fmt.Printf("mounting relocation mapping %q\n", d.relMapping)
```

Comment:
I believe this should be `fmt.Fprintf(os.Stdout...` or it should be removed.

Reply:
Oops, I forgot to delete that. Thanks for spotting this.